nanome/_internal/_network/_commands/_serialization/_integration.py

```diff
@@ -1,19 +1,18 @@
 from nanome._internal._util._serializers import _TypeSerializer
-from nanome._internal._network._commands._callbacks._commands_enums import _Hashes, _IntegrationCommands
+from nanome._internal._network._commands._callbacks._commands_enums import _Hashes, _Integrations
 from nanome._internal._integration import _serialization as Serializers
 
 class _Integration(_TypeSerializer):
     __integrations = {
-        _Hashes.IntegrationHashes[_IntegrationCommands.hydrogen_add]: Serializers._AddHydrogen(),
-        _Hashes.IntegrationHashes[_IntegrationCommands.hydrogen_remove]: Serializers._RemoveHydrogen(),
-        _Hashes.IntegrationHashes[_IntegrationCommands.structure_prep]: Serializers._StructurePrep(),
-        _Hashes.IntegrationHashes[_IntegrationCommands.calculate_esp]: Serializers._CalculateESP(),
-        _Hashes.IntegrationHashes[_IntegrationCommands.minimization_start]: Serializers._StartMinimization(),
-        _Hashes.IntegrationHashes[_IntegrationCommands.minimization_stop]: Serializers._StopMinimization(),
-        _Hashes.IntegrationHashes[_IntegrationCommands.export_locations]: Serializers._ExportLocations(),
```

Comment:
Did you mean to remove `export_locations`?

Reply:
huuuuuu, this is all very confusing hahaha. I'll check that today

Reply:
nice catch man!